languages/csharp/Bitwarden.Sdk/Bitwarden.Sdk.csproj

```diff
@@ -1,11 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <RootNamespace>Bitwarden.Sdk</RootNamespace>
-
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Is anything using unsafe on the c# side?

Reply:
Yes, `[LibraryImport] does, it generates code like this that has unsafe blocks:

```csharp
private static partial void run_command_async(string json, global::Bitwarden.Sdk.BitwardenSafeHandle handle, global::Bitwarden.Sdk.BitwardenLibrary.OnCompleteCallback on_completed_callback)
{
    byte* __json_native = default;
    nint __handle_native = default;
    System.IntPtr __on_completed_callback_native = default;
    // Setup - Perform required setup.
    global::System.Runtime.InteropServices.Marshalling.SafeHandleMarshaller<global::Bitwarden.Sdk.BitwardenSafeHandle>.ManagedToUnmanagedIn __handle_native__marshaller = new();
    scoped global::System.Runtime.InteropServices.Marshalling.Utf8StringMarshaller.ManagedToUnmanagedIn __json_native__marshaller = new();
    try
    {
        // Marshal - Convert managed data to native data.
        __on_completed_callback_native = on_completed_callback != null ? global::System.Runtime.InteropServices.Marshal.GetFunctionPointerForDelegate(on_completed_callback) : default;
        __handle_native__marshaller.FromManaged(handle);
        __json_native__marshaller.FromManaged(json, stackalloc byte[global::System.Runtime.InteropServices.Marshalling.Utf8StringMarshaller.ManagedToUnmanagedIn.BufferSize]);
        {
            // PinnedMarshal - Convert managed data to native data that requires the managed data to be pinned.
            __handle_native = __handle_native__marshaller.ToUnmanaged();
            __json_native = __json_native__marshaller.ToUnmanaged();
            __PInvoke(__json_native, __handle_native, __on_completed_callback_native);
        }

        // NotifyForSuccessfulInvoke - Keep alive any managed objects that need to stay alive across the call.
        global::System.GC.KeepAlive(on_completed_callback);
    }
    finally
    {
        // CleanupCallerAllocated - Perform cleanup of caller allocated resources.
        __handle_native__marshaller.Free();
        __json_native__marshaller.Free();
    }

    // Local P/Invoke
    [global::System.Runtime.InteropServices.DllImportAttribute("bitwarden_c", EntryPoint = "run_command_async", ExactSpelling = true)]
    static extern unsafe void __PInvoke(byte* __json_native, nint __handle_native, System.IntPtr __on_completed_callback_native);
}
```